public/static/docs/command-reference/update.md

```diff
@@ -69,3 +77,43 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
+
+Let's import a data artifact from an older commit from our
+[get started example repo](https://github.com/iterative/example-get-started) at
+first:
+
+```dvc
+$ dvc import --rev 72e0f1 git@github.com:iterative/example-get-started model.pkl
+Importing 'model.pkl (git@github.com:iterative/example-get-started)'
+-> 'model.pkl'
+```
+
+After this, the import stage (DVC-file) `model.pkl.dvc` is created.
+
+Let's try to run `dvc update` on the given stage file, and see what happens.
+
+```dvc
+$ dvc update model.pkl.dvc
+```
+
+There was no output at all, meaning, the `model.pkl` file was not updated. This
+is because, we tied the import stage with a `rev` that never changes (i.e.
+commit hash never changes). Therefore, it was not updated.
+
+Let's try to update it to the latest version by specifying revision to `master`
+branch.
+
+```dvc
+$ dvc update --rev master model.pkl.dvc
```

Comment:
let's try to not change semantics - from fixed commit to not following the branch

Reply:
Do you have any good examples in mind? I'm using this as a kind of a hack for being short in examples. I tried using the recent `aita_dataset` but, the dataset on `master` was too large (~117 MB) just for the demonstration (but, it's quite good in that I can change the --rev from `master` to `lightweight`, quite good example really).

Reply:
we can use tags at least? there are two tags in the example-get-started - baseline-experiment and bigrams-experiment

Reply:
tags are more or less static, and you have to use re-import or use `dvc update --rev` to move the latest version

what do you think?

Reply:
@shcheklein, i moved to tags. Showing example of moving between experiments is good, but, I feel importing from a not-changing revision and trying to move to a recent version of artifacts is also a good example worth showing off.

Reply:
@skshetry agreed, but we need a bit different approach then (we can't rely on a specific hash since it might change). Git clone repo use git to take the latest hash? (basically show how to take it?) .... or even use `dvc list` - may be a good idea for the list to show commit hashes?

Reply:
Seems like a similar question as https://github.com/iterative/dvc.org/pull/1007#discussion_r386716408:

> It's baiscaly the same as Example: Fixed revisions and updating to different revision in dvc import. Maybe we should just copy/paste it...